### PR TITLE
Story-845: Migrate to Maven Central - Releases 

### PR DESCRIPTION
**Pull request by JayasriR**

### CODE REVIEW

**Comment by ja6a-regnosys** [comment]
What problem did this change address?

**Comment by JayasriR** [comment]
> What problem did this change address?

Earlier, the releases were not deplpyed to maven central as the pom.xml did not have the necessary plugins configured to deploy. So this change enables migration of releases to maven central.

File: pom.xml [hunk 1]
```diff
@@ -448,7 +448,27 @@
...
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <version>${maven-javadoc-plugin.version}</version>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-source-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <artifactId>maven-deploy-plugin</artifactId>
```
Comment (ja6a-regnosys) [inline pom.xml:461]: where is the group?

File: pom.xml [hunk 2]
```diff
@@ -448,7 +448,27 @@
...
+                <version>${maven-deploy-plugin.version}</version>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-gpg-plugin</artifactId>
```
Comment (ja6a-regnosys) [inline pom.xml:466]: why do we have versions in some places but not other?

**Review by ja6a-regnosys — CHANGES_REQUESTED** [review]
please address the comment

**Review by ja6a-regnosys — APPROVED** [review]

**Pull request merged**